src/index.js

```diff
@@ -1,4 +1,13 @@
-// const refresh = document.getElementById('refresh-btn');
-// refresh.addEventListener('click', refreshEvent);
-// const submitBtn = document.getElementById('submit-btn');
-// submitBtn.addEventListener('click', addEvent);
+import './style.css';
+import submitEven from './modules/submit.js';
+import { refresh, refreshEvent } from './modules/reload.js';
+
+const refreshbtn = document.getElementById('refresh-btn');
+refreshbtn.addEventListener('click', refreshEvent);
+
+const submitBtn = document.getElementById('submit-btn');
+submitBtn.addEventListener('click', submitEven);
+
+window.onload = () => {
+  refresh();
+};
```

Comment:
- [ ] At page load, There are some errors appear in the console, Please fix them.